src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -67,6 +67,8 @@ public partial class Process : Component
 
         private static object s_createProcessLock = new object();
 
+        private bool _inputStreamAccessed;
```

Comment:
Wouldn't it be better to name it `_standardInputAccessed`? IMO it is better association it with the variable `_standardInput`.